nerfstudio/process_data/colmap_utils.py

```diff
@@ -460,6 +462,11 @@ def colmap_to_json(
     applied_transform[2, :] *= -1
     out["applied_transform"] = applied_transform.tolist()
 
+    # create ply from colmap
+    assert ply_filename.endswith(".ply"), f"ply_filename: {ply_filename} does not end with '.ply'"
+    create_ply_from_colmap(ply_filename, recon_dir, output_dir)
```

Comment:
There is a bug here, when creating ply from colmap, you need to apply `applied_transform` to the point cloud similar like here: https://github.com/nerfstudio-project/nerfstudio/blob/main/nerfstudio/data/dataparsers/colmap_dataparser.py#L354-L366

Otherwise, the camera poses will be in different world coordinate as point clouds. And the quality of training GS will be severely affected.